demo/csi/hetzner/volume-job.nomad

```diff
@@ -0,0 +1,37 @@
+job "example" {
+  datacenters = ["fsn1"]
+
+  group "cache" {
+    volume "test" {
+      type   = "csi"
+      source = "${volume_id}"
+    }
+
+    task "redis" {
+      driver = "docker"
+
+      config {
+        image = "redis:3.2"
+
+        port_map {
```

Comment:
This is using deprecated `port_map` syntax. Let's update this to a current version of Nomad's setup for networking.